### DISCUSSION TITLE
Default Enterprise username and password

### DISCUSSION THREAD

**Discussion by Evert-jvv** [ant-media/Ant-Media-Server#6710] [FAQs] [answered]
When launching the enterprise edition from the AWS AMI, it starts a server with a default account, what are the details for that default account? I remember when we first subscribed there was a note somewhere that provided the details.

**Comment by yashtandon113** [comment, marked as answer]
Hi @Evert-jvv 

If you have launched the Ant Media Server via AWS Marketplace image, the default username is the **JamesBond**, and the password is the instance ID of your EC2 instance, like **i-0b22a22eec53xxxx**.
​
Let me know if you have any questions.
